src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -461,6 +461,9 @@ well as menu structures (for main menu and popup menus).
          <shortcut refid="expandToMatching" value="Ctrl+Shift+E" if="org.rstudio.core.client.BrowseCap.isMacintosh()"/>
          <shortcut refid="expandToMatching" value="Ctrl+Shift+Alt+E" if="!org.rstudio.core.client.BrowseCap.isMacintosh()"/>
          <shortcut refid="splitIntoLines" value="Cmd+Alt+A"/>
+         <shortcut refid="showDocumentOutline" value="Ctrl+Alt+G"/>
```

Comment:
I think the command should be "toggleDocumentOutline" (it likely already is just want to make sure). Also think we should steal the Cmd+Shift+O shortcut from "Source File" and use it for the outline (I think it's okay to have no shortcut for Source File or to give it another more obscure shortcut if need be).